docs/user-guide/iam/ras/ras.md

```diff
@@ -0,0 +1,254 @@
+---
+slug: /user-guide/ras/resource-bundles
+title: RAS - Resource Bundles
+description: Learn how to manage resource level permissions with resource bundles
+last_update:
+  date: 09/19/2025
+  author: Soltan Garayev
+---
+
+import acorn from 'acorn';
+import { mdxExpression } from 'micromark-extension-mdx-expression';
+import Img from '@site/src/components/Img';
+
+# Resource Authorization System
+
+## Resource Bundle Management Overview
+
+The Resource Authorization System (RAS) in IOMETE provides a comprehensive way to organize and manage access to your data platform resources through **Resource Bundles**. Think of resource bundles as organized containers that group related resources together and control who can access them.
+
+RAS helps you:
+- **Organize resources** into logical groups for easier management
+- **Control access** by granting specific permissions to users and groups
+- **Maintain security** by ensuring only authorized users can access sensitive resources
+- **Simplify administration** through centralized permission management
+
+<Img src="/img/user-guide/iam/ras/bundle-list.png" alt="Resource Bundle List" maxWidth="800px" />
+
+## Understanding Resource Bundles
+
+### What are Resource Bundles?
+
+A resource bundle is a collection of related resources (called assets) with associated access permissions. Each bundle contains:
+
+- **Assets**: The actual resources like compute clusters, Spark jobs, storage configurations, and workspaces
+- **Permissions**: Specific access rights granted to users and groups for each type of resource
+- **Ownership**: Who manages and controls the resource bundle
+
+### Resource Bundle Ownership
+
+Bundles can be owned by:
+- **Individual Users**: Only the specific user can manage the resource bundle
+- **Groups**: Any member of the group (including nested group members) can manage the resource bundle
+
+## Creating Your First Resource Bundle
+
+Follow these steps to create a new resource bundle:
+
+1. **Navigate to Resource Bundle Management**
+   - Open the IOMETE Console
+   - Go to the **Other** section in Domain page
```

Comment:
lets remove this, this seems that there is some kind of action needs to be taken